opts/secret.go

```diff
@@ -53,10 +52,6 @@ func (o *SecretOpt) Set(value string) error {
 		case "source", "src":
 			options.SecretName = value
 		case "target":
-			tDir, _ := filepath.Split(value)
```

Comment:
Why are we fine with a target being a directory? This still doesn't make sense right?